packages/backend/src/database/migrations/20230602131116_validation-reset-current.ts

```diff
@@ -0,0 +1,51 @@
+import { Knex } from 'knex';
+
+const ValidationTableName = 'validations';
+
+export async function up(knex: Knex): Promise<void> {
+    if (!(await knex.schema.hasTable(ValidationTableName))) {
+        await knex.schema.createTable(ValidationTableName, (table) => {
```

Comment:
Only handle the case where we update the table. If the table isn't there something really bad happened.